nautobot/docs/models/extras/customfield.md

```diff
@@ -68,6 +68,15 @@ There are a number of available built-in filters for custom fields.
 
 Filtering on an object's list view follows the same pattern as [custom field filtering on the API](../../rest-api/filtering.md#filtering-by-custom-field).
 
+When using the ORM, you can filter on custom fields as follows (assuming a custom field of string type called 'site_code' was created for Site objects):
+
+```python
+from nautobot.dcim.models import Site
+
+all_sites = Site.objects.all()  # -> ['Raleigh', 'Charlotte']
+filtered_sites = Site.objects.filter(_custom_field_data__mycustomfield="US-NC-RAL42")  # -> ['Raleigh']
```

Comment:
Shouldn’t it be site_code? Based on the prior description

Reply:
That is of course correct - fixed.